app/components/footer.tsx

```diff
@@ -6,28 +6,22 @@ import { forwardRef, ReactNode } from "react";
 
 import Flex, { FlexProps } from "@/components/flex";
 import contentRoutes from "@/content-routes.json";
-import { BUILD_COMMIT, BUILD_GITHUB_REPO, BUILD_VERSION } from "@/domain/env";
+import { BUILD_VERSION } from "@/domain/env";
 import { useLocale } from "@/locales/use-locale";
 
 const Version = () => {
-  let commitLink = null;
-
-  if (BUILD_GITHUB_REPO && BUILD_COMMIT) {
-    commitLink = (
-      <>
-        (
-        <Link
-          color="primary"
-          href={`${BUILD_GITHUB_REPO}/commit/${BUILD_COMMIT}`}
-        >
-          {BUILD_COMMIT.substr(0, 7)}
-        </Link>
-        )
-      </>
-    );
-  } else if (BUILD_COMMIT) {
-    commitLink = `(${BUILD_COMMIT.substr(0, 7)})`;
-  }
+  const commitLink = (
+    <>
+      (
+      <Link
+        color="primary"
+        href={`https://github.com/visualize-admin/visualization-tool/blob/main/CHANGELOG.md`}
+      >
+        <Trans id="footer.version.change-log">Change Log</Trans>
```

Comment:
I'd use a `Changelog` convention, without a space (see how it's defined in the actual changelog: https://github.com/visualize-admin/visualization-tool/blob/main/CHANGELOG.md) 👀

Reply:
@bprusinowski what about other languages?

Reply:
Not 100% sure about that 😅 I think that `Changelog` is a bit of an international word, but maybe there are dedicated words in other languages that don't sounds weird (e.g. in Polish, I wouldn't use "rejestr zmian" for that, but keep "Changelog") 😅

Reply:
In German it would be `Änderungsprotokoll`, I personally think `Changelog` is better, is this our call?

Reply:
I'd say we go with `Changelog`, we can adjust this easily going forward 👍